com.github.sunsided.k8sfwd.yml

```diff
@@ -0,0 +1,40 @@
+---
+app-id: com.github.sunsided.k8sfwd
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: k8sfwd
+finish-args:
+  - "--share=network"
+  # - "--filesystem=xdg-config"
+  # - "--filesystem=home"
+  - "--filesystem=host"
+  - "--allow=devel"
```

Comment:
also why 'devel'. you don't even explain why.

Reply:
Devel is not on purpose. I am trying to sort out which requirements are needed to get the file access I needed and `devel` sounded a lot like "developer tools" - removed it. :)